requirements/frozen.py3.txt

```diff
@@ -218,6 +216,7 @@ pickleshare==0.7.5
 pika==0.12.0
 Pillow==5.3.0
 pluggy==0.6.0
+polarion-docstrings==0.15
```

Comment:
We can pickup `0.16`

Reply:
freeze_all script picked up this version